Assets/MixedRealityToolkit/UX/Scripts/Keyboard/KeyboardInputField.cs

```diff
@@ -6,7 +6,7 @@
 using UnityEngine.EventSystems;
 using UnityEngine.UI;
 
-namespace MixedRealityToolkit.UX.Keyboard
+namespace MixedRealityToolkit.UX.VirtualKeyboard
```

Comment:
If we've moved these to a new VirtualKeyboard namespace, the folder name should be renamed as well..